docs/usage/self-hosted-configuration.md

```diff
@@ -577,34 +577,35 @@ Otherwise, Renovate skips onboarding a repository if it finds no dependencies in
 
 ## onboardingPrTitle
 
-Similarly to `onboardingBranch`, if you have an existing Renovate installation and you change `onboardingPrTitle` then it's possible that you'll get onboarding PRs for repositories that had previously closed the onboarding PR unmerged.
+If you have an existing Renovate installation and you change the `onboardingPrTitle`: then you may get onboarding PRs _again_ for repositories with closed non-merged onboarding PRs.
+This is similar to what happens when you change the `onboardingBranch` config option.
```

Comment:
The orignal sentence is tough to rewrite, because it has a lot of conditions:

- Behavior is similar to `onboardingBranch`
- Sentence only applies to those with existing Renovate installation
- Only affects repositories with previously closed un-merged onboarding PRs (previously, closed, unmerged, onboarding... that's 4 things to remember)

At least now we have two sentences, to split one of the conditions (similar behavior)...

Open for suggestions to make this easier to understand! 😉